common/shlibs

```diff
@@ -2512,6 +2512,7 @@ libzimg.so.2 zimg-2.0.4_1
 libvapoursynth.so vapoursynth-R32_2
 libvapoursynth-script.so.0 vapoursynth-R32_1
 libtwolame.so.0 twolame-0.3.13_1
+libobs-scripting.so obs-22.0.3_2
```

Comment:
Is something using this ?

Reply:
Nope, not yet. but `xbps-src` was throwing warnings